src/backend/src/services/reimbursement-requests.services.ts

```diff
@@ -80,6 +92,86 @@ export default class ReimbursementRequestService {
     return createdReimbursementRequest;
   }
 
+  /**
+   * sends the pending advisor reimbursements to the advisor
+   * @param sender the person sending the pending advisor list
+   * @param saboNumbers the sabo numbers of the reimbursement requests to send
+   */
+  static async sendPendingAdvisorList(sender: UserWithTeam, saboNumbers: number[]) {
+    await validateUserIsPartOfFinanceTeam(sender);
+
```

Comment:
we should check that saboNumbers isn't an empty array

Reply:
smart